src/analysis/normed_space/linear_isometry.lean

```diff
@@ -228,20 +247,41 @@ namespace linear_isometry_equiv
 variables (e : E ≃ₛₗᵢ[σ₁₂] E₂)
 
 include σ₂₁
+
+lemma to_linear_equiv_injective : injective (to_linear_equiv : (E ≃ₛₗᵢ[σ₁₂] E₂) → (E ≃ₛₗ[σ₁₂] E₂))
+| ⟨e, _⟩ ⟨_, _⟩ rfl := rfl
+
+@[simp] lemma to_linear_equiv_inj {f g : E ≃ₛₗᵢ[σ₁₂] E₂} :
+  f.to_linear_equiv = g.to_linear_equiv ↔ f = g :=
+to_linear_equiv_injective.eq_iff
+
+instance : add_monoid_hom_class (E ≃ₛₗᵢ[σ₁₂] E₂) E E₂ :=
```

Comment:
Shouldn't this be an `add_equiv_class` instance?

Reply:
Yes, but that doesn't exist yet. This instance is better than no instance at all!

Reply:
Ooh wow, I somehow assumed that the equiv_class PR had landed two weeks ago already...